doc/new_spectrograph.rst

```diff
@@ -53,6 +53,12 @@ new spectrograph are as follows:
 #. Add the new module to the list imported by
    ``pypeit/spectrographs/__init__.py``.
 
+#. Generate a new Telescope object in (if new)
```

Comment:
Would it also be helpful to point them explicitly to this PR (#1179) if they want an example?

Reply:
done